docs/sql-migration-guide.md

```diff
@@ -22,6 +22,14 @@ license: |
 * Table of contents
 {:toc}
 
+## Upgrading from Spark SQL 3.5.1 to 3.5.2
+
+- Since 3.5.2, MySQL JDBC datasource will read TINYINT UNSIGNED as ShortType, while in 3.5.0 and 3.5.1, it was wrongly read as ByteType.
```

Comment:
`3.5.0 and 3.5.1, it was wrongly read as ByteType` part seems to conflict with the following section, `Upgrading from Spark SQL 3.5.0 to 3.5.1`'s ` TINYINT UNSIGNED as ByteType, while in Spark 3.5.0 and below, they were read as IntegerType` line.

Reply:
Just for my understanding, for `TINYINT UNSIGNED`, does Spark 3.5.0 read it as `ByteType` or `IntegerType`?

Reply:
Oops, it shall be `while in 3.5.1`